examples/artemis-transaction-consumer/artemis_transaction_consumer.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/artemis;
+import ballerina/io;
+
+public function main() returns error? {
+    // Create the Artemis listener with `autoCommitAcks` set to `false`.
+    // In this case the message acknowledgements will be committed only
+    // if the transaction block completes successfully.
+    artemis:Listener lis = new artemis:Listener({host:"localhost", port:61616});
+    // Create a `Consumer` object to make blocking receive calls
+    var consumer = lis.createAndGetConsumer({
+        queueName: "example"
+    });
+    if (consumer is artemis:Consumer) {
+        transaction {
+            // Makes a blocking receive call
```

Comment:
```suggestion
            // Makes a blocking receive call
```

+1 if "blocking receive call" is a known technical term. If not, we will have to reword this.

Reply:
changed to "synchronous receive"